playbooks/base/pre.yaml

```diff
@@ -22,10 +22,8 @@
     - import_role: name=ensure-output-dirs
       when: ansible_user_dir is defined
 
-    - name: Enable EPEL9
-      ansible.builtin.dnf:
-        name:
-          - epel-release
-          - epel-next-release
```

Comment:
next is recommended for the Stream, do you think you can easily apply it only to the c9s'?

Reply:
`epel-next-release` is a weak dependency of `epel-release` and `dnf` (which I hope the `package` module calls) installs weak dependencies by default.

Reply:
cool then :sunglasses: